src/core/QueryManager.ts

```diff
@@ -247,9 +252,8 @@ export class QueryManager<TStore> {
 
         (result: FetchResult<TData>) => {
           if (graphQLResultHasError(result) && errorPolicy === 'none') {
-            throw new ApolloError({
-              graphQLErrors: result.errors,
-            });
+            const graphQLErrors = mergeGraphQLErrors(result);
+            throw new ApolloError({ graphQLErrors });
```

Comment:
```suggestion
            throw new ApolloError({ graphQLErrors: mergeGraphQLErrors(result) });
```

[suuuuuuppperrrrrr nit] If you want to reduce to a single line, I find this readable as well.

Reply:
Resolved in https://github.com/apollographql/apollo-client/pull/10368/commits/31949b340454b176fba65b0f46d92afc23b3a735#diff-5fb8ad16cfbcb51bc035d32b5963734561c440b48417a1c2dbf80a16098be67bR255-R257.